src/script/lua_api/l_mapgen.cpp

```diff
@@ -754,6 +754,28 @@ int ModApiMapgen::l_get_mapgen_object(lua_State *L)
 }
 
 
+// get_spawn_level(x = num, z = num)
+int ModApiMapgen::l_get_spawn_level(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+
+	int x = luaL_checkinteger(L, 1);
+	int z = luaL_checkinteger(L, 2);
+	v2s16 pos2d = v2s16(x, z);
```

Comment:
unnecessary temporary variable

Reply:
Ok will do, considered it, did this for clarity.

Reply:
Sorry i see what you mean now, very siily =)

Reply:
Done.